src/lolex-src.js

```diff
@@ -170,12 +170,10 @@ function withGlobal(_global) {
     }
 
     function createDate() {
-        function ClockDate(year, month, date, hour, minute, second, ms) {
+        function createNativeDate(year, month, date, hour, minute, second, ms) {
             // Defensive and verbose to avoid potential harm in passing
             // explicit undefined when user does not pass argument
             switch (arguments.length) {
-                case 0:
```

Comment:
Keep this case. This is pretty easy to read and keeps all the logic that deals with the number of arguments in the same place, instead of spreading the logic around.

Reply:
Done